test/utils/skipLogic.js

```diff
@@ -294,30 +300,55 @@ tap.test('.skipLogic()', { autoend: true }, (t) => {
       }]
     }
 
-    skipLogicGroupCheck(scope, 'yes', check)
+    skipLogicGroupCheck(scope, check)
     t.notOk(scope.field.show)
 
-    check.logicGate = 'or'
-    skipLogicGroupCheck(scope, 'yes', check)
-    t.ok(scope.field.show)
-    skipLogicGroupCheck(scope, 'no', check)
+    scope.form.var2.$modelValue = 'yes'
+    skipLogicGroupCheck(scope, check)
     t.ok(scope.field.show)
 
-    check.logicGate = 'and'
-    check.group[1].value = 'yes'
-    skipLogicGroupCheck(scope, 'yes', check)
+    t.end()
+  })
+
+  t.test('skipLogicGroupCheck(): should match one depth group appropriately with "or" logicGate', (t) => {
+    const scope = {
+      field: {
+        settings: {
+          showhide: null
+        }
+      },
+      form: []
+    }
+
+    scope.form['var1'] = { $modelValue: 'yes' }
+    scope.form['var2'] = { $modelValue: 'no' }
+    scope.form['var3'] = { $modelValue: 'yes' }
+
+    const check = {
+      logicGate: 'or',
+      action: 'showhide',
+      group: [{
+        operand: '=',
+        value: 'yes',
+        variable: 'var1'
```

Comment:
probably declaring this before the form variable setters above would make this test case more useful.